x/concentrated-liquidity/internal/model/pool.go

```diff
@@ -15,20 +15,24 @@ var (
 	_ types.ConcentratedPoolExtension = &Pool{}
 )
 
-func NewConcentratedLiquidityPool(poolId uint64, denom0, denom1 string, currSqrtPrice sdk.Dec, currTick sdk.Int) (Pool, error) {
+func NewConcentratedLiquidityPool(poolId uint64, denom0, denom1 string, currSqrtPrice sdk.Dec, tickSpacing uint64) (Pool, error) {
+	// Order the initial pool denoms so that denom0 is lexicographically smaller than denom1.
```

Comment:
Do we have a test for this function? Let's make an issue if not please

Reply:
Unit test created